app/controllers/navigator/base_controller.rb

```diff
@@ -180,7 +180,7 @@ def build_folder_tree(folder, children)
       archived = params[:archived] == 'true'
       tree = fetch_projects(folder.parent_folder, archived).map { |i| project_serializer(i, archived) } +
              fetch_project_folders(folder.parent_folder, archived).map { |i| folder_serializer(i, archived) }
-      tree.find { |i| i[:id] == folder.code }[:children] = children
+      tree.find { |i| i[:id] == folder.code } && tree.find { |i| i[:id] == folder.code }[:children] = children
```

Comment:
Did you check why this issue appears at all? Theoretically we build tree level based on parent, and folder must exist in this level.

Reply:
@aignatov-bio I noticed there is a 500 error caused in this line when viewing tasks in archived state.
This happens when task project folder is active
So query to search project folders will not have the current task project folder in it.

Reply:
but then probably you will lost some folder will be missed in list. So if we inside archived task we still must see whole tree even if this archived